server/resp/src/main/java/org/infinispan/server/resp/json/JsonSetFunction.java

```diff
@@ -22,17 +20,22 @@
 import com.jayway.jsonpath.JsonPath;
 import com.jayway.jsonpath.PathNotFoundException;
 
+@ProtoTypeId(ProtoStreamTypeIds.RESP_JSON_SET_FUNCTION)
 public class JsonSetFunction
       implements SerializableFunction<EntryView.ReadWriteEntryView<byte[], JsonBucket>, String> {
    public static final String ERR_VALUE_CAN_T_BE_NULL = "value can't be null";
    public static final String ERR_PATH_CAN_T_BE_NULL = "path can't be null";
-   public static final AdvancedExternalizer<JsonSetFunction> EXTERNALIZER = new JsonSetFunction.Externalizer();
 
-   byte[] value;
-   byte[] path;
-   boolean nx;
-   boolean xx;
+   @ProtoField(1)
+   final byte[] value;
+   @ProtoField(2)
+   final byte[] path;
+   @ProtoField(3)
+   final boolean nx;
+   @ProtoField(4)
+   final boolean xx;
```

Comment:
not related to the change, but how am I suppose to know what `xx` and `nx` mean!?